CRM/Activity/Import/Parser/Activity.php

```diff
@@ -147,23 +159,11 @@ public function summary(&$values) {
 
     // For date-Formats
     $session = CRM_Core_Session::singleton();
-    $dateType = $session->get('dateTypes');
     if (!isset($params['source_contact_id'])) {
       $params['source_contact_id'] = $session->get('userID');
     }
     foreach ($params as $key => $val) {
-      if ($key === 'activity_date_time') {
-        if ($val) {
-          $dateValue = CRM_Utils_Date::formatDate($val, $dateType);
-          if ($dateValue) {
-            $params[$key] = $dateValue;
```

Comment:
I think removing this params assignment is ok. It gets passed on to isErrorInCustomData below but this field isn't custom data. In testing it still catches invalid dates.

And just noting this is now a silly loop, looping thru all the params just to find engagement_level. And the comments on line 160 and 172 no longer apply.